### PR TITLE
feat(defi): add support for uniswap and sushiswap

### PR DESCRIPTION
**Pull request by mullapudipruthvik**
Ticket- https://bitgoinc.atlassian.net/browse/BG-35078
We want to support top DEFI apps using our bitgo wallet.
For this PR I have added support for UniSwap and SushiSwap.

Major Changes:
1. src/eth/methods/methods.ts added support for input type typle which is required for UniSwapV3(Version 3) and is not supported earlier.
2.eth/config/instances.json added contract addresses.
3.Examples for each contract

### CODE REVIEW

File: eth/examples/Sushiswap/kashiBorrow.ts [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+  const amountWETH = '20000000000000000';
+  /*
+   * 24 ACTION_BENTO_SETAPPROVAL ParamNames user, _masterContract, approved, v, r, s. ABI encoding address, address, bool, uint8, bytes32, bytes32
+   * for information in v, r, s visit https://github.com/ethereum/EIPs/blob/master/EIPS/eip-155.md
+   * 5 ACTION_BORROW ParamNames amount, to. ABI encoding int256, address.
+   * 20 ACTION_BENTO_DEPOSIT ParamNames token, to, amount, share. ABI encoding IERC20, address, int256, int256
+   * 10 ACTION_ADD_COLLATERAL ParamNames share, to, skim. ABI encoding int256, address, bool
+   */
+  const actions = ['24', '5', '20', '10'];
```
Comment (karankurbur) [inline eth/examples/Sushiswap/kashiBorrow.ts:18]: unclear what values i would replace this with if i was going to copy this example, add more docs

**Comment by 0xJacobV** [comment]
LGTM. It might be helpful to include an example of providing/withdrawing liquidity to/from a pool since its a very common use case

File: eth/examples/Sushiswap/kashiBorrow.ts (hunk 1 above)
Comment (mullapudipruthvik) [reply eth/examples/Sushiswap/kashiBorrow.ts:37]: > unclear what values i would replace this with if i was going to copy this example, add more docs

I have added some examples, you can review now

**Comment by mullapudipruthvik** [comment]
> LGTM. It might be helpful to include an example of providing/withdrawing liquidity to/from a pool since its a very common use case

I have added remove liquidity examples.

**Review by karankurbur — CHANGES_REQUESTED** [review]
there are no helper functions to create inputs into methods, my comments not addressed

File: eth/examples/Uniswap/createLiquidityPoolV3.ts [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+  const fee = '3000';
+
+  const tickLower = '-887220';
+  const tickUpper = '887220';
+  const amount0Desired = '14789861394480592';
+  const amount1Desired = '49999999999999998309';
+  const amount0Min = '14753024820732246';
+  const amount1Min = '49874843358150006642';
+  const deadline = '1634103700';
```
[the remark this answers is not in the document]
Comment (mullapudipruthvik) [reply eth/examples/Uniswap/createLiquidityPoolV3.ts]: Hey karan,
I have added helper function where required, rest of the variables are not constants.

**Review by karankurbur — APPROVED** [review]

**Review by karankurbur — APPROVED** [review]

**Pull request merged**